Formula/c/cloudsmith.rb

```diff
@@ -0,0 +1,136 @@
+class Cloudsmith < Formula
```

Comment:
maybe better to use CloudsmithCli?

see https://repology.org/project/cloudsmith-cli/versions